docs/specification/appendix.md

```diff
@@ -33,6 +33,10 @@ The Channel `message_encoding` field describes the encoding for all messages wit
 
 - `message_encoding`: [`flatbuffer`](https://google.github.io/flatbuffers/)
 
+### capnproto
+
+- `message_encoding`: [`capnproto`](https://capnproto.org/)
```

Comment:
> Do we need to specify packed or not packed

Good question. 

> Cap’n Proto messages are often “packed” with a simple compression algorithm that deflates zero-value bytes.

It does seem that one has to select either packed or unpacked. I'm not sure if we should first serialize the leading byte of `data` to indicate pack-edness

Reply:
In our use case, I don't think we would encourage users to pack, since we already compress the file, and unpacking will cause an additional unpacking step.

I am for not supporting packed messages.